### ISSUE TITLE
value-list: screen reader only instructions appears with keyboard interaction when dragEnabled

### ISSUE THREAD

**Issue by anveshmekala** [Esri/calcite-components#5690]
### Actual Behavior

When the user interacts with the `drag-enabled` value-list component, screen reader helper text is displayed.

### Expected Behavior

Screen reader only instructions should be hidden.

### Reproduction Sample

https://codepen.io/ANVESHMEKALA/pen/zYaBeKZ

### Reproduction Steps

1. In the codepen, try to tab into the draghande of any value-list item.
2.  Press space key.
3. observe a text being displayed at the top of the list with instructions.

### Reproduction Version

0.96

### Regression?

beta.86

### Esri team

N/A

**Comment by jcfranco** [comment]
Express-triaged for this milestone. cc @benelan @geospatialem @brittneyzwolfer

**Comment by jcfranco** [comment]
Verified with the latest release: https://codepen.io/jcfranco/pen/ExRZqbG?editors=100
